tests/go.sum

```diff
@@ -829,6 +829,8 @@ github.com/onsi/gomega v1.10.2/go.mod h1:iN09h71vgCQne3DLsj+A5owkum+a2tYe+TOCB1y
 github.com/op/go-logging v0.0.0-20160315200505-970db520ece7/go.mod h1:HzydrMdWErDVzsI23lYNej1Htcns9BCg93Dk0bBINWk=
 github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver v0.24.0 h1:lepRkNbEJDWuH1neuc+qPqPh1ys6QSVAMZiKne5tvrk=
 github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver v0.24.0/go.mod h1:tR/dOYoq6Uky6SYCcqAHA/dZU4savuTNRnXpZ4pWMlQ=
+github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver v0.24.1-0.20210409203624-f1ee616c4d90 h1:cUViOe5xsV7GyGwkU1adx+Vt7T89ygduCcM6Fij5L/U=
```

Comment:
Can you run `go mod tidy` in the `tests` directory? That seems to remove the earlier entry for `jmxreceiver` in go.sum.

Reply:
Done